usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -1690,38 +1774,122 @@
         ),
         "federal_account": OrderedDict(
             [
-                ("owning_agency_name", "treasury_account__federal_account__parent_toptier_agency__name"),
-                ("reporting_agency_name", "reporting_agency_name"),  # Column is annotated in account_download.py
+                ("owning_agency_name", "treasury_account__funding_toptier_agency__name"),
```

Comment:
This is a Federal Account level download so I think it makes sense to leave it as the previous query_path so that it rolls up correctly o the Federal Account `owning_agency_name` instead of the `treasury_account` value.

Reply:
This could also hold true for a few of the fields that were previously marked with `# Column is annotated in account_download.py`.

Reply:
I see what happened here. Reverted